kemeny_sourcemash.tex

```diff
@@ -68,17 +68,17 @@ \subsection{Development}
 
 \section{Technical Description}
 
-Sourcemash was built to be used as an RSS feed aggregator, but at its core is a categorization algorithm. We will break down the technical description into these two components: the categorization algorithm and the RSS reader that incorporates the algorithm to allow users to get parse through the news faster.
+Sourcemash was built to be used as an RSS feed aggregator, but at its core is a news categorization engine. We will break down the application's technical description into two components: the categorization algorithm and the RSS reader that incorporates the algorithm to allow users to parse through their news faster.
```

Comment:
Don't think we want to say news here. The categorization can generalize to any webpage/text.